runtime/vm/ObjectFieldInfo.hpp

```diff
@@ -342,11 +342,16 @@ class ObjectFieldInfo {
 	{
 		U_32 fieldDataStart = 0;
 		if (!isContendedClassLayout()) {
-			fieldDataStart = getSuperclassFieldsSize();
+#ifdef J9VM_OPT_VALHALLA_VALUE_TYPES
```

Comment:
I think this might read better with an |= in the VT case.

Reply:
C++ doesn't have logical assignments, so I did `|=`.

Reply:
Is that strictly valid? I suppose a bitwise OR does give the correct result.

Reply:
Maybe "x = x || y" instead...

Reply:
done